### DISCUSSION TITLE
Cannot Access Network Manager

### DISCUSSION THREAD

**Discussion by spflanze** [openzfs/zfs#13564] [Q&A]
I followed the instructions at:
https://openzfs.github.io/openzfs-docs/Getting%20Started/Ubuntu/Ubuntu%2022.04%20Root%20on%20ZFS.html
The only modifications I made were to add partitions for SLOG, and L2ARCH. I succeeded, and I am now able to boot Ubuntu on it.

I opted to have network access managed graphically, and followed the instructions in Step 7.2. Now the computer does not have network access. I verified NetworkManager is installed just as would be expected by using the command `nmcli --version` which returns version 1.36.6. Yet I do not see its icon in the task bar in the upper right corner of the screen.

I attempted to follow the instructions at:
https://www.configserverfirewall.com/ubuntu-linux/ubuntu-network-manager/
to start up NetworkManager. I got this error:
```
sudo netplan generate
/etc/netplan/01-newtork-manager-all.yaml:3:8: Invalid YAML: mapping values are not allowed in this context:
version: 2
       ^
```
What do I need to do to get the Network Manager working?

**Comment by rincebrain** [comment]
That doesn't really sound like a ZFS-related issue; you should probably ask on Ubuntu-related forums.
